sycl/doc/extensions/experimental/sycl_ext_oneapi_non_uniform_groups.asciidoc

```diff
@@ -521,7 +521,8 @@ the fixed-size-group.
 ----
 range_type get_group_range() const;
 ----
-_Returns_: A `range` representing the number of fixed-size-groups.
+_Returns_: A `range` representing the number of fixed-size-groups in a parent
+group.
```

Comment:
Maybe something like:
```suggestion
_Returns_: A `range` representing the number of fixed-size-groups created by
partitioning the parent group with `Size`.
```

I'm being pedantic, but the number of fixed-size-groups in a parent group might be a concept that some people struggle with.  If somebody partitions a group into fixed-size-groups of size 1 and again into fixed-size-groups of size 2, those two partitionings don't know anything about one another.